pheno005/phenotype/demographics.json

```diff
@@ -0,0 +1,75 @@
+{
+	"participant_id": {
+		"Description": "OpenNeuro ID of the subject."
+	},
+	"session_id": {
+		"Description": "ID assigned to participant's visit.",
+		"Levels": {
+			"baseline": "First imaging and phenotype data collection visit.",
+			"followup": "Second phenotype data collection visit."
+		}
+	},
+	"run_id": {
```

Comment:
I wonder whether it should be `run_id`, `run_index`, or just `run`? See: https://bids-specification.readthedocs.io/en/stable/glossary.html#index-common_principles. I would be inclined to have it as just `run`.

Reply:
I don't have strong feelings so I'll go with `run` on this one.